README.md

```diff
@@ -1,2 +1,13 @@
 # Steeltoe InitializrConfiguration
+
 Configuration for [Steeltoe InitializrWeb](https://github.com/SteeltoeOSS/InitializrWeb)
+
+### Build Templates (locally)
+
+`./bin/build`
+
+### Run Docker
+
+`docker build -t initializr-config .`
```

Comment:
The Docker image currently produced from this project is `initializr-izrs`.  It's not documented anywhere, but the reason is that the Docker image's sole purpose is to serve up IZR template bundles.  Eventually (soon?) the API will get the templates directly from GitHub and the `initializr-izrs` will be obsoleted.